bin/crew

```diff
@@ -1922,5 +1925,14 @@ Signal.trap('INT') do
 end
 
 load_json
+if @device[:last_update_check].blank?
+  @device.store('last_update_check', '0')
+  @last_update_check = 0
+  save_json(@device)
+else
+  @last_update_check = @device[:last_update_check]
+  crewlog("The last update was #{time_difference(@last_update_check, Time.now.to_i)} ago.")
+  puts "It has been more than #{CREW_UPDATE_CHECK_INTERVAL} day#{CREW_UPDATE_CHECK_INTERVAL < 2 ? '' : 's'} since crew was last updated. Please run 'crew update'".lightpurple if Time.now.to_i - @last_update_check > (CREW_UPDATE_CHECK_INTERVAL * 3600 * 24)
```

Comment:
Can't we just check the last time the git repository was fetched?

Reply:
Ah!

I guess we could use `stat -c %Y .git/FETCH_HEAD` ?